### ISSUE TITLE
signInWithPopup() with google provider does not redirect to google.

### ISSUE THREAD

**Issue by jifalops** [firebase/polymerfire#81]
### Description

Using `signInWithPopup()` with the google provider opens a popup on my domain and never redirects to google.

### Expected outcome
The popup would redirect to google and show a login screen, which is what happens with the notes app codelab.

### Actual outcome
The popup opens with the url: 

https://myapp.firebaseapp.com/__/auth/handler?apiKey=x&appName=myapp&authType=signInViaPopup&providerId=google.com&scopes=profile&eventId=959678822&v=3.2.0

and does not redirect.

### Steps to reproduce

Create a project using `polymer init`. Firebase setup: 

```
<firebase-app
      name="myapp"
      auth-domain="myapp.firebaseapp.com"
      database-url="https://myapp.firebaseio.com"
      api-key="x"
      storage-bucket="firebase-myapp.appspot.com"></firebase-app>
    <firebase-auth
      id="auth"
      app-name="myapp"
      provider="google"
      signed-in="{{signedIn}}"
      user="{{user}}"
      on-error="onAuthError"></firebase-auth>
```
And a button calls `signInWithPopup()`. I get the same outcome whether on my local machine (via `polymer serve`) or after using `firebase deploy`. 

`firebase version` => 3.0.4 
`polymer --version` => 0.12.0

### Browsers Affected
I have only tested on chrome.

**Comment by midesweb** [comment]
This problem is resolved in this thread: https://github.com/Polymer/polymer-cli/issues/290

**Comment by jifalops** [comment]
The navigateFallbackWhitelist trick isn't working for me, locally or deployed. I should mention that even when I'm testing locally, signInWithPopup() opens my firebaseapp index page in a popup, which doesn't seem right. But if I change to `<firebase-app auth-domain="localhost"` the app is never initialized.

**Comment by jifalops** [comment]
Okay it turns out I was blocking a third party cookie from google that was preventing the redirect if that makes sense. The navigateFallbackWhitelist entry was probably also needed. I had been developing on my chromebook and decided to try the code on my linux desktop and it worked. Going back to the chromebook I had already enabled cookies on the popup, but I had to explicitly allow the one from google since they were third party.
